arangod/Pregel/Status/Status.h

```diff
@@ -34,17 +35,34 @@
 #include <Pregel/Common.h>
 
 namespace arangodb::pregel {
+
+template<typename T>
+auto add(std::optional<T> a, std::optional<T> b) -> std::optional<T> {
```

Comment:
Why not const reference? If in later use `a` and/or `b` are larger than just numbers, it would make no sense to copy them.